src/pages/article.tsx

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { Link } from 'gatsby';
+import Articles from '../components/Landing/ArticlesSection';
+import ArticleImg from '../images/ArticleImg.svg';
+import ArticleImgLarge from '../images/ArticleImgLarge.svg';
+import Fleche from '../assets/icons/fleche.svg';
+import Video from '../images/video.svg';
+import Reseaux from '../assets/icons/reseaux.svg';
+
+import Navbar from '../components/Navbar';
+import Footer from '../components/Footer';
+
+function ArticlePage() {
+  const videos = [{ icon: Video,
+    videoName: 'video',
+  }];
+  const articlesImg = [
+    {
+      img: ArticleImg,
+      index: 1,
+    },
+    {
+      img: ArticleImg,
+      index: 2,
+    },
+
+  ];
+  const articlesImg2 = [
+    {
+      img: ArticleImg,
+      index: 1,
+    },
+    {
+      img: ArticleImg,
+      index: 2,
+    },
+    {
+      img: ArticleImg,
+      index: 3,
+    },
+  ];
+  return (
+    <>
+      <Navbar />
+      <main>
+        <section className="text-white">
+          <div className="text-white flex font-clash px-24 md:px-52 py-4">
+            <Link
+              to="/"
+              className="text-white flex font-clash text-xl items-center "
+            >
+              <Fleche />
+              {'   '}
+              blog
+            </Link>
+          </div>
+          <div className="px-24 md:px-52 flex-col py-6 ">
+            <h1 className="md:text-5xl  w-1/2 font-clash text-2xl max-md:w-full">
+              Titre lorem ipsum dolor si amet consectetur adipiscing elit.
+            </h1>
+            <div className="flex-col items-center py-12 md:flex">
+              <div className="bg-black-light rounded-2xl md:absolute md:left-12">
+                <div className="flex">
+                  <div className=" rounded-2xl z-0 border-white border md:border-1 mr-2 px-2">
+                    tech
+                  </div>
+                  <div className=" rounded-2xl z-0 border-white border md:border-1 mr-2 px-2">
+                    design
+                  </div>
+                </div>
+                <section className=" w-4/6  pb-8 max-lg:w-full ">
+                  <div>
+                    <p className="text-purple">writer</p>
+                    <p>Jean Michel Ipsum</p>
+                  </div>
+                  <div>
+                    <p className="text-purple">reading time</p>
+                    <p>8min</p>
+                  </div>
+                  <div>
+                    <p className="text-purple">share on </p>
+                    <Reseaux />
+                  </div>
+                </section>
+              </div>
+              <img src={ArticleImgLarge} alt="ArticleImg" />
+            </div>
+            <h2 className="text-xl py-12 font-semibold max-md:text-lg">
```

Comment:
All the content after this cannot contain any classes it should all be "infered" by the html tags and the custom css for this page. 
This is because when we will make it dynamic you will receive a payload like this : 

```json

{
  "data": {
    "articles": [
      {
        "content": {
          "markdown": "It is important to take care of the patient, to be followed by the patient, but it will happen at such a time that there is a lot of work and pain. Nibh nisl the condiment that is poisoned by the condiment of life. The fans of the game don't need anything else. A large and well-equipped vestibule. Unless someone is an eleifend than a coach**. How can anyone do that? Unless someone is an eleifend who is a coach of the life of the shooters. He pursues and triggers until he hates the kids for the rest of his life. Duis ultricies lake but ugly tincidunt. Tortor,**but the author of the pot now has some fear of this course. For I have to drink from the foul taste of football.\n\n-   The time of life that the children do not for some.\n-   The whole of life just needs a great warmth.\n-   Or what element the pillow is not even.\n\n\n-   The main course of the estate of the employee was targeted by the snipers. But now the element of life will be taken care of. He does not need the quiver and the ultrice to adorn the bronze. Or an elit selecrelic mauris pellentesque pulvinar. Bed urn two valley valley valley\n\n\n\n_It does no_[_t require any keyboard players._](https://google.com \"https://google.com\")__To drink beer at the bed. Some expected to drink the bow of life, the element of life will be taken care of now. Laughter at the time of basketball is not expected. I read a lot of ultris in the course. Tellus rutrum tellus pellentesque eu tincidunt tortor aliquam nulla.\n\n\n\n    And he expected hunger and ugly want. Diam volutpat is convenient, but it needs to be done. He wants the arrows to be chased as pure as a pillow. Who can put chocolate in his mouth?\n\n![Logo_long_hd.png](https://eu-central-1-shared-euc1-02.graphassets.com/clv4w1nk80b3307w2881491kq/clv928zla0k4j07un7ght6lfu \"Logo_long_hd.png\")\n\n\n",
          "html": "<p>It is important to take care of the patient, to be followed by the patient, but it will happen at such a time that there is a lot of work and pain. Nibh nisl the condiment that is poisoned by the condiment of life. The fans of the game don&#39;t need anything else. A large and well-equipped vestibule. Unless someone is an eleifend than a coach<strong>. How can anyone do that? Unless someone is an eleifend who is a coach of the life of the shooters. He pursues and triggers until he hates the kids for the rest of his life. Duis ultricies lake but ugly tincidunt. Tortor, </strong>but the author of the pot now has some fear of this course. For I have to drink from the foul taste of football. </p><ul><li><div>The time of life that the children do not for some.</div></li><li><div> The whole of life just needs a great warmth. </div></li><li><div>Or what element the pillow is not even. </div></li></ul><ul><li><div>The main course of the estate of the employee was targeted by the snipers. But now the element of life will be taken care of. He does not need the quiver and the ultrice to adorn the bronze. Or an elit selecrelic mauris pellentesque pulvinar. Bed urn two valley valley valley</div></li></ul><p></p><p><em>It does no</em><a target='_blank' title=\"https://google.com\" href=\"https://google.com\"><em>t require any keyboard players.</em></a><em> </em>To drink beer at the bed. Some expected to drink the bow of life, the element of life will be taken care of now. Laughter at the time of basketball is not expected. I read a lot of ultris in the course. Tellus rutrum tellus pellentesque eu tincidunt tortor aliquam nulla. </p><p></p><pre><code>And he expected hunger and ugly want. Diam volutpat is convenient, but it needs to be done. He wants the arrows to be chased as pure as a pillow. Who can put chocolate in his mouth?</code></pre><img src=\"https://eu-central-1-shared-euc1-02.graphassets.com/clv4w1nk80b3307w2881491kq/clv928zla0k4j07un7ght6lfu\" alt=\"Logo_long_hd.png\" title=\"Logo_long_hd.png\" width=\"680\" height=\"340\" /><p></p>"
        },
        "coverPicture": {
          "fileName": "9070079.jpg",
          "url": "https://eu-central-1-shared-euc1-02.graphassets.com/clv4w1nk80b3307w2881491kq/clv92kfn4100607unhrcwafxs"
        }
      }
    ]
  }
}
```
So there will be no classes or anything in it